src/arrayMethodJoin.js

```diff
@@ -4,8 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let result = '';
+    const LENGTH_OF_ARRAY = this.length;
```

Comment:
I'm not entirely sure, but it's not easier to immediately use this.length, instead of a separate variable LENGTH_OF_ARRAY.

Reply:
If you want to create variable for this, name it in camelCase